frontend/src/components/widgets/DataFrame/columns/cells/DateTimeCell.tsx

```diff
@@ -0,0 +1,209 @@
+/**
+ * Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react"
+
+import styled from "@emotion/styled"
+
+import {
+  CustomCell,
+  CustomRenderer,
+  drawTextCell,
+  GridCellKind,
+  ProvideEditorCallback,
+  TextCellEntry,
+} from "@glideapps/glide-data-grid"
+
+export const StyledInputBox = styled.input`
```

Comment:
Should this be in a styled-components? Cool that we added this!

Reply:
Hmm, it could be. But in this case, I wanted to keep it a bit more self-contained since, eventually, this is supposed to be removed and replaced by the updated version in glide.

Reply:
Sounds good. I don't have any problem with this being in there then.